Makefile

```diff
@@ -0,0 +1,87 @@
+#! /usr/bin/make -f
+# -*- makefile -*-
+# ex: set tabstop=4 noexpandtab:
+
+default: help all
+	sync
+
+project=castanets
+url?=https://github.com/Samsung/castanets
+
+dir=out/Default
+exe_basename=chrome
+exe=${dir}/${exe_basename}
+depot_tools_dir?=/usr/local/opt/depot_tools
+PATH:=${PATH}:${depot_tools_dir}
+gclient_file?=${CURDIR}/../.gclient
+gn?=gn
+maintainer?=p.coval@samsung.com
+pkglicense?=BSD-3-clause and others
+pkgversion?=$(shell git describe --tag | tr '-' '+' || echo 0.0.0)
+pkgrelease?=0~${USER}0
+pkgsummary?=Web engine distributed among multiple devices
+debian_requires?= \
+ libasound2, \
+ libgconf-2-4, \
+ libgtk-3-0, \
+ libnss3, \
+ libxss1, \
+ libgl1 \
+ # EOL
+
+%: help
+	@echo "# log: $@"
+	sync
+
+help:
+	@echo "# log: Check 'README.md' for more details"
+	@echo "# PATH=${PATH}"
+	gn --version || echo "# TODO: install gn"
+
+all: ${exe}
+	ls $<
+
+${gclient_file}: build/create_gclient.sh
```

Comment:
`gclient sync` is always performed, and it takes too long time.
I think we need an option to skip it.

Reply:
@rzr Will this be updated?